extension-client/src/api/index.ts

```diff
@@ -242,8 +247,9 @@ export async function pullLatest(data: any) {
 }
 
 export async function versions() {
+    
     const schematicsService = await auth.getSchematicsService();
-
+    
```

Comment:
remove unnecessary new lines

Reply:
We formatted the code and removed extra spaces